src/index.ts

```diff
@@ -21,7 +21,7 @@ export class Proxy {
           .then(options => this.transport.request(options))
           .then(rawRequest => rawRequest.pipe(res))
           .catch((err) => {
-            const status = err.statusCode || 500;
+            const status = (err.statusCode !== undefined) ? err.statusCode : 500;
```

Comment:
Here's something that would be a touch more bulletproof:

```suggestion
            const status = (typeof err.statusCode !== 'undefined') ? err.statusCode : 500;
```